src/Components/Queue/stubs/queue.php

```diff
@@ -2,6 +2,19 @@
 
 return [
 
+    /*
```

Comment:
Can we remove this from the config, and document that people can actually override this here?

Reply:
Remove it from the configuration stubs by default and just mention it in the docs? Or have I misunderstood?

Reply:
Yeah - just default this option to false on the internals of the framework, and if people want to change it they can do it here.

Reply:
Sounds good, I'll open a PR soon. :+1: